requirements/base.in

```diff
@@ -14,4 +14,4 @@ rapidfuzz
 pandas
 scipy
 pypdfium2
-pdfminer-six==20240706
\ No newline at end of file
+pdfminer-six>=20240706
```

Comment:
doesn't this always resolve to the same package, currently the latest one? just remove the pin altogether?

Reply:
Yes, but in future the following versions will be mandatory.

Langchain follows the evolution of versions for [PDFMinerLoader](https://python.langchain.com/docs/integrations/document_loaders/pdfminer/) and it will not be possible to combine it with unstructured.
The final objective of my series of [Pull Request](https://github.com/langchain-ai/langchain/pull/28970) for LangChain is to be able to choose the parser for each case, with `PDFRouterLoader`. This means being able to have several parsers at the same time. Freezing a version prevents this.

no problem for you to do it yourself.
Take this opportunity to publish a new version, and adjust, in unstructured, extra-pdf-image.in, with the new version.

Reply:
Just an fyi, @cragwolfe, but the initial pin for `pdfminer-six` was added just a few weeks ago when removing `pdfplumber` [(here)](https://github.com/Unstructured-IO/unstructured-inference/commit/625255a810730100c09d5cfc6ff9618b26a6c368#diff-ffb9a8aa35f8e58dd818a724713720da25a2847e19d36b4bbf5da7d5744415e0) to maintain required packages used by scripts to pass CI. It sounds like it's a workaround we might want to fix. For easy reference, though, here's [extra-pdf-image.in](https://github.com/Unstructured-IO/unstructured/blob/main/requirements/extra-pdf-image.in)

@pprados, I'm confused what you're saying needs to be adjusted in `extra-pdf-image.in`. The `pdfminer.six` version isn't pinned there, so it should be install the latest as @cragwolfe mentioned, right?

Reply:
@cragwolfe, if this PR looks good to you, can you approve my duplicate PR [here](https://github.com/Unstructured-IO/unstructured-inference/pull/412)? The CI failure in this one is due to a secret that's needed in CI itself and not just particular tests, so I'm unsure how to fix that for contributor PRs at the moment.